templates/index.html

```diff
@@ -29,4 +29,5 @@
   {% include "takeovers/_tackling_iot.html" %}
   {% include "takeovers/_de-vmware-to-os.html" %}
```

Comment:
You can remove this takeover according to the schedule:

https://docs.google.com/spreadsheets/d/1MaFd-ZHWpRVjIP9Sj_dOCIGOtl-GJYbrXewt5EG4m80/edit#gid=564832475